PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<ul><li>Lines 1-2 in this block: `took a medicine` should be `took medicine`</li><li>As above, printing 10 lines at the top and bottom of the dataframe is a lot. Consider printing just five total rows.</li><li>I would lower your sample size from `q=1000` to `q=100` to make sampling much quicker. Currently it takes a ridiculous amount of time to sample a single chain, and people may give up if they're following this tutorial and see that they have to wait 750 seconds per chain.</li></ul>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Made the changes</p>

Reply:
I just noticed that taking due to `q = 100`, the chains did not converge. Should we use a bigger value for `q`

Reply:
It might be worth sampling a longer chain instead of using a really big sample -- try running the sampler for more iterations and then bringing `q` back up if that doesn't work.

Reply:
I tried using `q = 100` and 2000 iterations but it did not converge. Using `q = 100` and 2500 iterations worked well and the chains converged. For each chain, the time for each chain was about 110-140 seconds. Do you think this is reasonable?

Reply:
Yeah, and it's much less time than using `q=1000`. That's about par for a model of this type.